tests/matrix_free/create_mesh.h

```diff
@@ -1,7 +1,17 @@
-//-----------------------  create_mesh.h  -----------------------------
-//    Version: $Name$
```

Comment:
How many times do we have to nuke these? I feel like I have tried to delete all of them at least three times now.

Reply:
It's the zombie string!